### PR TITLE
Support b3 in soil download templates

### PR DESCRIPTION
**Pull request by biyeun**
@emord / anyone

### CODE REVIEW

**Comment by biyeun** [comment]
@czue or @dannyroberts :)

File: corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% load i18n %}
```
Comment (benrudolph) [inline corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html]: does b3 drop support for dropbox sync?

**Comment by dannyroberts** [comment]
Oops, didn't realize @benrudolph's comment was maybe blocking

File: corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html (hunk 1 above)
Comment (biyeun) [inline corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html]: I believed we intentionally dropped dropbox sync from exports... @dimagi/product do you remember that to be the case?

Comment (benrudolph) [inline corehq/ex-submodules/soil/templates/soil/partials/bootstrap3/download_in_progress.html]: that could be true but i believe this modal is used in a variety of places other than that, notably media download

**Pull request merged**